src/auth/basic/SSPI/basic_sspi_auth.cc

```diff
@@ -112,6 +113,7 @@ main(int argc, char **argv)
     char wstr[HELPER_INPUT_BUFFER];
     char username[256];
     char password[256];
+    char sep = '\0', junk = '\0';
```

Comment:
One declaration per line, please. Also, initializing these new variables here is misleading because they are not reset during the loop and inconsistent because `username` and `password` are _not_ initialized _here_. Please move these declarations to where we call sscanf() instead.

Reply:
done in [b7155e9](https://github.com/squid-cache/squid/pull/2160/commits/b7155e9c32f15548b02d4c3d907a9d93c66df875)